### ISSUE TITLE
Error (AttributeError: 'module' object has no attribute 'operations') running "studio run" 

### ISSUE THREAD

**Issue by shobhalakshminarayana** [studioml/studio#327]
When I run cmd `studio run examples/keras/train__mnist_keras.py` I get below error.  What is the resolution for this error?
```Traceback (most recent call last):
  File "/sml/bin/studio-run", line 6, in <module>
    exec(compile(open(__file__).read(), __file__, 'exec'))
  File "/studio/studio/scripts/studio-run", line 3, in <module>
    runner.main()
  File "/studio/studio/runner.py", line 412, in main
    max_duration=runner_args.max_duration,
  File "/studio/studio/experiment.py", line 135, in create_experiment
    for pkg in pip.operations.freeze.freeze():
AttributeError: 'module' object has no attribute 'operations'
```

**Comment by analystanand** [comment]
I am also same issue 
`Traceback (most recent call last):
  File "/Users/parmanandsahu/test_studio_ml/bin/studio-run", line 3, in <module>
    runner.main()
  File "/Users/parmanandsahu/test_studio_ml/lib/python3.6/site-packages/studio/runner.py", line 412, in main
    max_duration=runner_args.max_duration,
  File "/Users/parmanandsahu/test_studio_ml/lib/python3.6/site-packages/studio/experiment.py", line 135, in create_experiment
    for pkg in pip.operations.freeze.freeze():
AttributeError: module 'pip' has no attribute 'operations' `

**Comment by karlmutch** [comment]
At a first blush this looks to be because of an issue with the version of pip being used, StudioML is not compatible with pip 10.  Can you downgrade to 9.0.3 and see if this goes away.

**Comment by karlmutch** [comment]
Hi Shobha,  Did this problem resolve itself with the downgrade to pip 9.0.3?  Thanks

**Comment by karlmutch** [comment]
Closed due to inactivity

**Comment by shobhalakshminarayana** [comment]
@karlmutch Yes, issue got resolved with pip 9.0.3.
